go.mod

```diff
@@ -16,13 +16,13 @@ require (
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.32.1
 	github.com/spf13/cobra v1.4.0
-	google.golang.org/grpc v1.40.0
-	k8s.io/api v0.24.3
-	k8s.io/apimachinery v0.24.3
-	k8s.io/client-go v0.24.3
-	k8s.io/component-base v0.24.3
-	k8s.io/component-helpers v0.24.3
-	k8s.io/klog/v2 v2.60.1
+	google.golang.org/grpc v1.47.0
+	k8s.io/api v0.25.0
+	k8s.io/apimachinery v0.25.0
+	k8s.io/client-go v0.25.0
+	k8s.io/component-base v0.25.0
+	k8s.io/component-helpers v0.25.0
+	k8s.io/klog/v2 v2.70.1
 	k8s.io/kubernetes v1.24.3
```

Comment:
Any reason why this one is not updated to 1.25?

Reply:
for some reason it was not picked up.. have corrected now.